tests/baselines/reference/destructuringControlFlow.js

```diff
@@ -73,8 +73,8 @@ function f3(obj) {
 function f4() {
     var _a, _b;
     var x;
-    (x = 0..x); // Error
-    (x = 0["x"]); // Error
+    (x = (0).x); // Error
+    (x = (0)["x"]); // Error

```

Comment:
This isn't incorrect but those extra parentheses could be avoided if I'd detect the access style~